linux/xkb/symbols/3l

```diff
@@ -78,3 +78,109 @@ xkb_symbols "cros" {
         symbols[Group1] = [ Tab ]
     };
 };
+
+
+// This is an alternative implementation, inspired by the original.
+// The main differences are:
+// 1. This layout implements just what appears in
+// https://github.com/jackrosenthal/threelayout/blob/master/docs/standard_three.png
+// (the original defines but does not document a bunch of greek letters, etc.)
+// 2. This layout uses CapsLock as the Cur modifier.
+// 3. This layout uses the Shift keys as the Sym modifiers.
+// 4. This layout uses the key just to the left of Enter
+// (the Sym key in the original layout) as "a funny kind of CapsLock":
+// - pressing it once capitalizes the next character
+// - pressing it twice acts as a regular CapsLock
+// - instead of double press, one can hold down either CapsLock (aka Cur)
+// or Shift (aka Sym) while pressing the "funny CapsLock" to get the
+// "regular CapsLock" behaviour
+// 5. In this layout, Escape is on Cur-Space as well as on Sym-Space
+partial alphanumeric_keys modifier_keys
+xkb_symbols "alt" {
```

Comment:
Is there a better name than "alt" we can use?

Reply:
I think the two main characteristics of this layout are that it uses Shift keys as Sym modifiers, and moves Cur modifier to CapsLock.  So maybe "shift_as_sym" for the name?

Reply:
Sounds good, but lets use hyphens instead of underscores. ;)